google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/AbstractMockServerTest.java

```diff
@@ -96,8 +97,18 @@ public abstract class AbstractMockServerTest {
                   .build())
           .setMetadata(SELECT_COUNT_METADATA)
           .build();
+  public static final com.google.spanner.v1.ResultSet UPDATE_RETURNING_RESULTSET =
+      com.google.spanner.v1.ResultSet.newBuilder()
+          .setStats(ResultSetStats.newBuilder().setRowCountExact(1))
+          .setMetadata(
+              ResultSetMetadata.getDefaultInstance()
+                  .toBuilder()
+                  .setRowType(StructType.getDefaultInstance()))
```

Comment:
This doesn't seem right. A DML statement with a `RETURNING` clause would return a result set with at least one column.

Reply:
Modified it to set row type to a StructType with 1 field (so that result set has 1 column).